### ISSUE TITLE
Merge project/adaptivemd to project/adaptivemd_gpu

### ISSUE THREAD

**Issue by itomaldonado** [radical-cybertools/radical.pilot#1584]
@andre-merzky I took care of a lot of the features already, but there is a specific feature that I was not *able* to get right.

Specifically the feature is the one introduced by branch `fix/issue_1466` (to support environment variables on ORTE)

The problem was that the conflicts are too large on very low-level files (e.g. `mpiexec.py`) which I sadly have no experience... Throughout this week I tried to tackle the merge it but I had to quickly revert because RP would not run properly.

This ticket is to track that progress.

**Comment by andre-merzky** [comment]
Hey Manuel - I'd be happy to help with that!  The env changes are indeed all over the place, and make merges difficult.  That's on my radar anyway though, as the GPU branche is being merged into devel right now, showing the very same conflicts.

Let me try to resolve those conflicts later today, once that is done, you merge should be simpler...

**Comment by andre-merzky** [comment]
Manuel, `project/adaptivemd` is very close to `devel`, and merges cleanly to `devel`.  `feature/GPU` is also about to be merged to `devel`.  So instead of doing very similar merges in two places, I would propose to properly merge `project/adaptivemd` into `devel`, and then leave the conflict resolution to the merge of `feature/gpu`.  Do you see a problem with that?

**Comment by andre-merzky** [comment]
Please have a look at `project/adaptivemd_gpu_am` - this contains the result of the 3-way merge (`devel`, `feature/gpu`, `project/adaptivemd`.

**Comment by jrossyra** [comment]
Just wanted to ping on this issue.

**Comment by iparask** [comment]
These branches have been deleted. As a result I think this is resolved. Closing as such
